src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
> I might even argue that we shouldn't have made Color an interpretation at all, since it's not really geometric...but that can wait for another Cortex version...

Yeah, so potentially we would even remove it from here [0] and have it return None/Numeric to begin with (or rename that function to not include "geometric" in the name)?

[0] https://github.com/mattigruener/gaffer/blob/26fe8bc7510c18e4e6f8f72658e575cdfc5de655/src/GafferImage/OpenImageIOAlgo.cpp#L75